roles/docker/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+docker_image: seravo/wordpress
```

Comment:
Maybe unnecessary? These are unlikely to change and the code is easier to read if there is less variables and these values could be embedded right where they are used.

Reply:
Removed roles/docker/defaults & hard-coded these values.